homeassistant/components/apcupsd/sensor.py

```diff
@@ -72,12 +76,14 @@
         name="UPS Battery Voltage",
         native_unit_of_measurement=UnitOfElectricPotential.VOLT,
         device_class=SensorDeviceClass.VOLTAGE,
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     "bcharge": SensorEntityDescription(
         key="bcharge",
         name="UPS Battery",
         native_unit_of_measurement=PERCENTAGE,
         icon="mdi:battery",
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
Couldn't this be `SensorDeviceClass.BATTERY`?

Reply:
Ah, missed this. You're right! This is now added 👍